plugins/vpp/l3plugin/route_config.go

```diff
@@ -170,7 +180,7 @@ func (plugin *RouteConfigurator) ModifyRoute(newConfig *l3.StaticRoutes_Route, o
 		return err
 	}
 
-	plugin.log.Infof("Route %v -> %v modified", oldConfig.DstIpAddr, oldConfig.NextHopAddr)
+	plugin.log.Infof("Route %v -> %v modified", oldConfig.DstIpAddr, newConfig.DstIpAddr)
```

Comment:
This info log is not really consistent with one on line 156. It was describing `DstIpAddr -> NextHopAddr` and not the actual modification.